yabeda-puma-plugin.gemspec

```diff
@@ -21,9 +21,11 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency "yabeda", "~> 0.2"
+  spec.add_runtime_dependency "yabeda", "~> 0.5.0"
+  spec.add_runtime_dependency "yabeda-prometheus"
```

Comment:
Not everyone uses Prometheus (there is also Datadog adapter, for example). And even if they does, not necessarily they use official client.

So this dependency should be _optional_ and removed from gemspec.

```suggestion
```